pato-base.obo

```diff
@@ -11,18 +11,22 @@ subsetdef: relational_slim "Relational slim: types of quality that require an ad
 subsetdef: scalar_slim "Scalar slim"
 subsetdef: value_slim "Value slim"
 default-namespace: quality
+idspace: dc http://purl.org/dc/elements/1.1/ 
+idspace: oboInOwl http://www.geneontology.org/formats/oboInOwl# 
+idspace: terms http://purl.org/dc/terms/ 
+idspace: terms1 http://www.geneontology.org/formats/oboInOwl#http://purl.org/dc/terms/ 
```

Comment:
yikes

Reply:
Fixed in the PR https://github.com/pato-ontology/pato/issues/573. GH Action changed this PR to update the imports, so I'll merge it to trigger a new release.